demo/index_filters.R

```diff
@@ -0,0 +1,29 @@
+
+library(RXapian)
+
+# preparing arguments for xapian_index()
+ID_NUMBER <-c("1974-100",
+              "1938-371",
+              "1985-438")
```

Comment:
Is it worth pulling these into a separate data file, so you don't need to duplicate them for every example?